internal/config/config.go

```diff
@@ -25,49 +27,55 @@ type Config struct {
 
 // Validate opens and processes the given yaml file, and catches errors in the process
 func Validate(filePath string) (Config, error) {
-	validationErr := errors.New("config file validation failed: ")
+	const validationErr = "config file validation failed: %w"
```

Comment:
Let's not prefix errors, it's the job of the parent

Reply:
done